README.md

```diff
@@ -25,6 +25,55 @@ cargo build --release
 ./target/release/cairo-rs-run tests/support/fibonacci_compiled.json
 ```
 
+## Running a Cairo Program with Cairo-rs
```

Comment:
Isn't this wrong? These are not instructions to run a Cairo program, but to run a function in  a Cairo program while passing its arguments